scripts/sbt-aws-cli/sbt_aws_cli/main.py

```diff
@@ -0,0 +1,447 @@
+            
+            if 'error' in status:
+                if status['error'] == 'authorization_pending':
+                    print('Authorization pending, checking again in a few seconds...')
+                elif status['error'] == 'slow_down':
+                    print('Polling too frequently, slowing down...')
+                    interval += 5
+                elif status['error'] == 'expired_token':
+                    print('Device code expired. Please try again.')
+                    return
+                elif status['error'].startswith('HTTP Error'):
+                    print('Retrying...')
+                else:
+                    print(f'Unexpected error: {status["error"]}')
+                    return
+            elif 'access_token' in status:
+                print('Authorization successful')
+                os.environ['ACCESS_TOKEN'] = status['access_token']
+                os.environ['REFRESH_TOKEN'] = status['refresh_token']
+                return
+            
+            attempts += 1
+
+        print('Authorization timed out. Please try again.')
+
+@app.command()
+def configure(
+    control_plane_stack: str = typer.Argument(..., help="Name of the Control Plane CloudFormation stack"),
+    client_id: str = typer.Argument(..., help="Device Cognito Client ID for authentication"),
+    fqdn: str = typer.Argument(..., help="Fully Qualified Domain Name for the control plane"),
+    control_plane_api_endpoint: str = typer.Argument(..., help="API endpoint for the control plane"),
+    cognito_domain: str = typer.Argument(..., help="Cognito domain for authentication"),
+    debug: bool = typer.Option(False, help="Enable debug mode")
+):
+    """
+    Configure the SBT CLI with Control Plane stack information and generate credentials.
+    """
+    client_secret = typer.prompt("Device Cognito Client Secret for authentication", hide_input=True)
+
+    if debug:
+        print("Configuring with:")
+        print(f"CONTROL_PLANE_STACK_NAME: {control_plane_stack}")
+        print(f"CLIENT_ID: {client_id}")
+        print(f"CLIENT_SECRET: {client_secret}")
+        print(f"FQDN: {fqdn}")
+        print(f"CONTROL_PLANE_API_ENDPOINT: {control_plane_api_endpoint}")
+        print(f"cognito_domain: {cognito_domain}")
+    
+    generate_credentials(client_id, client_secret, fqdn, debug)
+
+    config_data = {
+        "CONTROL_PLANE_STACK_NAME": control_plane_stack,
+        "CONTROL_PLANE_API_ENDPOINT": control_plane_api_endpoint,
+        "COGNITO_DOMAIN": cognito_domain,
+        "CLIENT_ID": client_id,
+        "CLIENT_SECRET": client_secret,
+        "FQDN": fqdn,
+        "ACCESS_TOKEN": os.getenv('ACCESS_TOKEN'),
+        "REFRESH_TOKEN": os.getenv('REFRESH_TOKEN')
+    }
+
+    with open(CONFIG_FILE, 'w') as config_file:
+        json.dump(config_data, config_file)
+
+    if debug:
+        print(f"Configuration saved to {CONFIG_FILE}")
+
+def source_config():
```

Comment:
i would recommend changing this name. "source" is something that is bash specific, as far as I know.

Reply:
How about `initialize_config`